features/registry/OWNERS

```diff
@@ -1,5 +1,9 @@
 approvers:
+  - xiuwang
 
 reviewers:
-- zhouying7780
-- pruan-rht
+  - pruan-rht
+  - wewang58
+  - wzheng1
+  - xiuwang
+  - zhouying7780
```

Comment:
I suppose pruan-rht and zhouying7780 can be removed here to reduce any noise to them : )

Reply:
@wzheng1 Once this PR got merged  you have the power to update that.

Reply:
Let me know if you would like me to update it here, or you would like to try/verify the process later.

Reply:
If it is convenient for you, feel free to update here directly. Thanks!

Reply:
@wzheng1 Updated

Reply:
Thanks!